Use the private core library shared by the BlueZ daemons and client.

--- a/configure.ac
+++ b/configure.ac
@@ -82,7 +82,7 @@
 				 ])


-PKG_CHECK_MODULES(GLIB, glib-2.0 >= 2.36)
+PKG_CHECK_MODULES(GLIB, fplinux-bluez-glib-2.0 >= 2.88.1)
 PKG_CHECK_MODULES(DBUS, dbus-1 >= 1.10)

 AC_ARG_WITH([dbusconfdir], AS_HELP_STRING([--with-dbusconfdir=DIR],
